test/SymbolGraph/Relationships/Synthesized/SuperclassImplementation.swift

```diff
@@ -3,15 +3,24 @@
 // RUN: %target-swift-symbolgraph-extract -module-name SuperclassImplementation -I %t -pretty-print -output-dir %t
 // RUN: %FileCheck %s --input-file %t/SuperclassImplementation.symbols.json
 
+// This test references code that has been removed. The implementation that synthesized superclass
+// methods was inconsistent; it failed to generate symbols for superclasses from another module.
+// From a symbol-relation perspective, the `inheritsFrom` relation from a subclass to its superclass
+// still exists, which already implies that all of the superclass's methods are available on the
+// subclass. The synthesized methods for subclasses were removed to provide consistency between
+// superclasses from the same module and those from a different one. If the implementation is
+// brought back, ensure that it consistently adds synthesized methods for superclasses from
+// different modules.
+
 public class Base {
   public init() {}
   public func foo() {}
 }
 
 public class Derived: Base {
-  // CHECK-DAG: "precise": "s:24SuperclassImplementation4BaseC3fooyyF::SYNTHESIZED::s:24SuperclassImplementation7DerivedC"
+  // CHECK-NOT: "precise": "s:24SuperclassImplementation4BaseC3fooyyF::SYNTHESIZED::s:24SuperclassImplementation7DerivedC"
```

Comment:
Should we check that an `inheritsFrom` relationship exists instead?

Reply:
The `inheritsFrom` relationship is already tested in the ["SymbolGraph/Relationships/InheritsFrom"](https://github.com/apple/swift/blob/286d22b2e6e282ef08dc810580c1ea94aac9ab08/test/SymbolGraph/Relationships/InheritsFrom.swift) test.